mne/datasets/utils.py

```diff
@@ -523,7 +527,7 @@ def has_dataset(name):
 
     Returns
     -------
-    has : bool
+      : bool
```

Comment:
By convention we always supply names for the returns even though it's not strictly necessary (and IIRC when you don't, you shouldn't have the `:`), please revert